doc/tutorial/introduction/index.md

```diff
@@ -153,6 +154,8 @@ a lot of parameters for the alignment computation, for instance score functions,
 you wish to compute a traceback or not. The configurations have their own namespace seqan3::align_cfg and can
 be combined via the logical OR operator (`|`) for building combinations. Check out the alignment tutorial if you want to learn more.
 
+Note: In general, we give `using namespace seqan3;` a miss, to show which parts are from SeqAn3 and which arn't. But for better readabiliy we use a `using` declaration for the namespace member `_dna5`. You can give it a miss, if you work with the overall namespace or declare it in lines 20 and 21 directly.
```

Comment:
Please reduce the line length to 120 characters.

Reply:
_to give a miss_ is colloquial language. Please use _omit_ or something similar (2x).

Reply:
~arn't~ => aren't

Reply:
~readabiliy~ => readability

Reply:
Instead of `Note:` you can use doxygen's `\note` command as in the line below.